packages/editor/src/components/post-actions/actions.js

```diff
@@ -1196,13 +1121,12 @@ export function usePostActions( postType, onActionPerformed ) {
 				  ! isPattern &&
 				  duplicatePostAction
 				: false,
-			! isTemplateOrTemplatePart && renamePostAction,
-			isTemplateOrTemplatePart && renameTemplateAction,
+			renamePostAction,
 			isPattern && exportPatternAsJSONAction,
-			isTemplateOrTemplatePart && resetTemplateAction,
-			! isTemplateOrTemplatePart && restorePostAction,
-			isTemplateOrTemplatePart && deleteTemplateAction,
-			! isTemplateOrTemplatePart && permanentlyDeletePostAction,
+			isTemplateOrTemplatePart ? resetTemplateAction : restorePostAction,
+			isTemplateOrTemplatePart
+				? deleteTemplateAction
+				: permanentlyDeletePostAction,
```

Comment:
Sidenote: The way we build the `actions` array here is difficult to parse. Maybe we should convert it into separate if conditions.

Example:

```js
if ( condition ) {
    actions.push( action );
}
```

Reply:
The order matters for UI for now and would need multiple `if` conditions. So I didn't do it here, even though I thought about it too. Will check it in follow ups where more actions will be consolidated.